### PR TITLE
Implementation for Ping model

### PR DESCRIPTION
**Pull request by ANISH-GOTTAPU**
Implementation for Ping model

### CODE REVIEW

File: snappi_ixnetwork/ping.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                if req_type == "ipv4":
+                    if src_name not in v4_names:
+                        msg = (
+                            src_name
+                            + """ is not available in the configured v4 interface names """
+                            + str(v4_names)
+                        )
+                        raise Exception(msg)
+                    ip_obj = (
+                        self._api._ixnetwork.Topology.find()
```
Comment (alakendu) [inline snappi_ixnetwork/ping.py:45]: It will better to find with some attribute (say Name) to optimize it.

File: snappi_ixnetwork/ping.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                        .DeviceGroup.find()
+                        .Ethernet.find()
+                        .Ipv6.find(Name=src_name)
+                    )
+                self._api.info("Sending ping to %s" % dst_ip)
+                ping_status = ip_obj.SendPing(DestIP=dst_ip)
+                for reply in ping_status:
```
Comment (alakendu) [inline snappi_ixnetwork/ping.py:66]: probably this is a corner case. But check command execution failure due to some exception

**Review by alakendu — APPROVED** [review]

File: snappi_ixnetwork/ping.py (hunk 1 above)
Comment (ANISH-GOTTAPU) [reply snappi_ixnetwork/ping.py:45]: ok

File: snappi_ixnetwork/ping.py (hunk 2 above)
Comment (ANISH-GOTTAPU) [reply snappi_ixnetwork/ping.py:66]: ok

**Pull request merged**